### ISSUE TITLE
TkClaimTest.java:39-41: Write more test cases for a...

### ISSUE THREAD

**Issue by 0pdd** [zerocracy/farm#761]
The puzzle `489-3f92a3b1` from #489 has to be resolved:

https://github.com/zerocracy/farm/blob/40954edd2a528ae70b121238d8035d4fb3e844c4/src/test/java/com/zerocracy/tk/project/TkClaimTest.java#L39-L41

The puzzle was created by amihaiemil on 07-Mar-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#761](https://github.com/zerocracy/farm/issues/761) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#761](https://github.com/zerocracy/farm/issues/761) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1211564362); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by g4s8** [comment]
@amihaiemil looks like this task is depend on #762 so I should start it after it, right?

**Comment by amihaiemil** [comment]
@g4s8 As far as I understand, no, because #762 only concerns the ``claims.xsl``, UI stuff. This task here concerns the backend TkClaim, which is already able to return the ``children`` element in a claim.

**Comment by g4s8** [comment]
@amihaiemil ok, thanks.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
The puzzle `489-3f92a3b1` has disappeared from the source code, that's why I closed this issue.

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/761#issuecomment-389122824))

@g4s8 This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#761](https://github.com/zerocracy/farm/issues/761) is [in scope](http://www.zerocracy.com/policy.html#14) for 2mon
  * The role is `DEV`
  * The job is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8) for [14days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @g4s8[/z](https://www.0crat.com/u/g4s8) will get 50% of their hourly rate for this job
  * Some equity will be [vested](http://www.zerocracy.com/policy.html#37) on completion
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23761%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@yegor256 this job was not removed from scope after close, can you do it manually?

**Comment by yegor256** [comment]
@0crat out

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @g4s8[/z](https://www.0crat.com/u/g4s8), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/761#issuecomment-390406290))

@yegor256 The job [#761](https://github.com/zerocracy/farm/issues/761) is now out of scope

**Comment by g4s8** [comment]
@ypshenychka this bug was fixed in #998 PR

**Comment by g4s8** [comment]
@ypshenychka BTW what do you think about #1029 ?

**Comment by g4s8** [comment]
@ypshenychka QA rules has changed:
> Ticket must have references to all pull requests where the problem was actually fixed;

so it's ok now to reference PR via GitHub links: https://github.com/zerocracy/farm/issues/761#ref-pullrequest-320706735

**Comment by ypshenychka** [comment]
@g4s8 I think that reference is indeed enough. Thanks for an update!

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
